FULL_HELP_DOCS.md

```diff
@@ -495,6 +495,18 @@ stellar contract invoke ... -- --help
 * `--instructions <INSTRUCTIONS>` — Number of instructions to simulate
 * `--build-only` — Build the transaction and only write the base64 xdr to stdout
 * `--sim-only` — Simulate the transaction and only write the base64 xdr to stdout
+* `--send <SEND>` — Whether or not to send a transaction
+
+  Default value: `if-write`
+
+  Possible values:
+  - `if-write`:
+    Only send transaction if there are ledger writes, otherwise return simulation result
+  - `no`:
+    Do not send transaction, return simulation result
+  - `yes`:
+    Always send transaction
```

Comment:
```suggestion
  - `if-write`:
    Send the transaction to the network if there are ledger writes or events published, otherwise output the simulation result and do not send to the network
  - `no`:
    Do not send the transaction to the network, and output the simulation result
  - `yes`:
    Send the transaction to the network, and output the transaction result
```